www/include/configuration/configNagios/formNagios.ihtml

```diff
@@ -264,7 +264,15 @@
                                     {include file="file:$centreon_path/www/include/common/templates/clone.ihtml" cloneId="broker" cloneSet=$cloneSet}
 				</td>
 			</tr>
-			<tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="event_broker_options">&nbsp;{$form.event_broker_options.label}</td><td class="FormRowValue">{$form.event_broker_options.html}</td></tr>
+            <tr class="list_two">
+                <td class="FormRowField">
+                    <img class="helpTooltip" name="event_broker_options">
+                    &nbsp;{$form.event_broker_options.label}
+                </td>
+                <td class="FormRowValue flex-row-wrap">
+                    {$form.event_broker_options.html}
+                </td>
+            </tr>
```

Comment:
indentation seems bad, but in fact it's ok :)
I moved tab into spaces for those lines